src/components/modal/addSrtModal/AddSrtModal.tsx

```diff
@@ -0,0 +1,290 @@
+}: AddSrtModalProps) {
+  const ingests = useIngests();
+
+  const [ingestUuid, setIngestUuid] = useState<string>('');
+  const [ingestName, setIngestName] = useState<string>('');
+  const [mode, setMode] = useState<SelectOptions>('Listener');
+  const [localIp, setLocalIp] = useState<string>('0.0.0.0');
+  const [localPort, setLocalPort] = useState<number>(1234);
+  const [remoteIp, setRemoteIp] = useState<string>('127.0.0.1');
+  const [remotePort, setRemotePort] = useState<number>(1234);
+  const [latency, setLatency] = useState<number>(120);
+  const [name, setName] = useState<string>('My SRT source');
+  const [passphrase, setPassphrase] = useState<string>();
+  const [isNameError, setIsNameError] = useState<boolean>(false);
+  const [isIngestNameError, setIsIngestNameError] = useState<boolean>(false);
+  const [srtPayload, setSrtPayload] = useState<SrtSource>({
+    mode: 'listener',
+    local_ip: localIp,
+    local_port: localPort,
+    remote_ip: remoteIp,
+    remote_port: remotePort,
+    latency_ms: latency,
+    name: name,
+    passphrase: passphrase
+  });
+
+  useEffect(() => {
+    if (ingestName !== '') {
+      const selectedIngest = ingests.find(
+        (ingest) =>
+          ingest.name.toLowerCase() === ingestName.toLowerCase().trim()
+      );
+
+      if (selectedIngest) {
+        setIngestUuid(selectedIngest.uuid);
+      }
+    }
+  }, [ingestName, ingests]);
+
+  useEffect(() => {
+    setSrtPayload({
+      mode: mode === 'Listener' ? 'listener' : 'caller',
+      local_ip: localIp || undefined,
+      local_port: localPort || undefined,
+      remote_ip: mode === 'Caller' ? remoteIp : undefined,
+      remote_port: mode === 'Caller' ? remotePort : undefined,
+      latency_ms: latency || undefined,
+      name: name,
+      passphrase: passphrase || undefined
+    });
+  }, [
+    mode,
+    localIp,
+    localPort,
+    remoteIp,
+    remotePort,
+    latency,
+    name,
+    passphrase
+  ]);
+
+  useEffect(() => {
+    if (isNameError) {
+      if (name.length !== 0 || name !== '') {
+        setIsNameError(false);
+      }
+    }
+  }, [isNameError]);
+
+  const handleCloseModal = () => {
+    setIsNameError(false);
+    setIsIngestNameError(false);
+    onAbort();
+  };
+
+  const handleCreateSrtSource = () => {
+    if (!name) {
+      setIsNameError(true);
+    }
+    if (ingestName === '') {
+      setIsIngestNameError(true);
+    }
+    if (!name || ingestName === '') {
+      return false;
+    }
+    if (!isIngestNameError && !isNameError) {
+      onConfirm(ingestUuid, srtPayload);
+      onAbort();
+    }
+    setIsIngestNameError(false);
+  };
+
+  const handleModeChange = () => {
+    setMode((prevMode) => (prevMode === 'Listener' ? 'Caller' : 'Listener'));
+    setIsIngestNameError(false);
+  };
+
+  useEffect(() => {
+    if (ingestName === t('inventory_list.select_ingest') || '') {
+      setIsIngestNameError(true);
+    } else {
+      setIsIngestNameError(false);
+    }
+  }, [ingestName]);
```

Comment:
```suggestion
  useEffect(() => {
    setIsIngestNameError([t('inventory_list.select_ingest'), ''].includes(ingestName))
  }, [ingestName]);
```

Reply:
In this case || "" does nothing. It is the same as || false. Maybe you meant: ingestName === t('inventory_list.select_ingest') || ingestName === "". In any case the suggested code should do the trick.